includes/kaltura_derivatives.inc

```diff
@@ -113,10 +107,7 @@ function islandora_video_streaming_kaltura_upload($object, $force = false) {
       if (file_put_contents($temp_name, json_encode($streaming)) === FALSE) {
         $messages = array(
           'message' => t('Failed to create temporary file @filename for object @pid when uploading to Kaltura.'),
-          'message_sub' => array(
```

Comment:
`@pid` is used here in 'message', but you've removed it from 'message_sub'.  These are interrelated.